op-service/txmgr/txmgr.go

```diff
@@ -76,6 +76,9 @@ type TxManager interface {
 	// the order of nonce increments.
 	SendAsync(ctx context.Context, candidate TxCandidate, ch chan SendResponse)
 
+	// ChainID returns the chain this tx-manager is connected to
+	ChainID() eth.ChainID
```

Comment:
It's not totally clear to me *why* a TxMgr now has to support chainID, though you do mention adding it. Why couldn't the faucet just have had a mapping from ChainID:TxMgr ?

Reply:
Discussed (as per other comment), repeating for any other reviewer/reader: the chainID is retrieved by the tx-manager, and we check if it's the expected chain, to not accidentally run a faucet for the wrong chain.